sympy/stats/crv_types.py

```diff
@@ -341,8 +350,37 @@ def pdf(self, x):
         alpha, beta = self.alpha, self.beta
         return x**(alpha - 1) * (1 - x)**(beta - 1) / beta_fn(alpha, beta)
 
-    def sample(self):
-        return random.betavariate(self.alpha, self.beta)
+    def _sample_python(self, size):
+        if size == ():
+            return random.betavariate(self.alpha, self.beta)
+        return [random.betavariate(self.alpha, self.beta) for i in range(size)]
```

Comment:
what if `size` is multidimensional? For example, you want to generate a matrix?

Reply:
multidimensional `size` is handled by all `exporting` libraries, for `_sample_python` I have to write code.

Reply:
Well... we have created `ArrayComprehension` at the beginning of this summer. It's meant to handle these kinds of cases.

Reply:
Right! Thanks!!

Reply:
`ArrayComprehension` with `lambda`? `ArrayComprehension(lambda: random.betavariate(1, 1), ...)` ?